api/stakeholder.go

```diff
@@ -125,7 +125,7 @@ func (h StakeholderHandler) Delete(ctx *gin.Context) {
 		h.deleteFailed(ctx, result.Error)
 		return
 	}
-	result = h.DB.Select(clause.Associations).Delete(m)
```

Comment:
Rely on cascade delete.
This can be dangerous because it will delete everything associated with the stakeholder with is not always correct.